inst/sql/sql_server/concept_plausible_gender.sql

```diff
@@ -13,18 +13,19 @@ plausibleGender = @plausibleGender
 **********/
 
 
-SELECT num_violated_rows, CASE WHEN denominator.num_rows = 0 THEN 0 ELSE 1.0*num_violated_rows/denominator.num_rows END AS pct_violated_rows
+SELECT num_violated_rows, CASE WHEN denominator.num_rows = 0 THEN 0 ELSE 1.0*num_violated_rows/denominator.num_rows END AS pct_violated_rows, 
+  denominator.num_rows as num_denominator_rows
 FROM
 (
 	SELECT COUNT_BIG(*) AS num_violated_rows
 	FROM
 	(
-		SELECT @cdmTableName.* 
-		FROM @cdmDatabaseSchema.@cdmTableName
-			INNER JOIN @cdmDatabaseSchema.person
-			ON @cdmTableName.person_id = person.person_id
-		WHERE @cdmFieldName = @conceptId
-		AND person.gender_concept_id <> {@plausibleGender == 'Male'} ? {8507} : {8532} 
+		SELECT A.* 
```

Comment:
Would recommend aliases that were more descriptive than A/B.

Reply:
Latest commit uses more descriptive aliases